src/cc/bcc_syms.h

```diff
@@ -65,8 +66,21 @@ int bcc_symcache_resolve_name(void *resolver, const char *module,
                               const char *name, uint64_t *addr);
 void bcc_symcache_refresh(void *resolver);
 
+struct mod_search {
```

Comment:
I feel this should probably be an internal struct (`bcc_syms.h` is user-facing header). You can just re-define it in the Unit Test.

Reply:
moved it to `bcc_syms.cc` and redefined in unit tests